src/TransactionTestsFiller/ttNonce/TransactionWithHighNonce64Minus1Filler.json

```diff
@@ -3,6 +3,7 @@
     {
         "expectException" : 
         {
+            ">=Frontier": "NonceMax"
```

Comment:
This new error also has to be added to retesteth config.

Reply:
@winsvega should it be `TR_NonceMax` ?

Reply:
Added https://github.com/ethereum/retesteth/pull/156 for this.